### ISSUE TITLE
django-allauth

### ISSUE THREAD

**Issue by snowman2** [tethysplatform/tethys#615]
I saw the OAuth stuff in the release (nice work, BTW). I figured I would mention [django-allauth](https://github.com/pennersr/django-allauth) in case you haven't heard of it and it might be useful. Feel free to close this issue if you have already considered this option.

**Comment by swainn** [comment]
Hey @snowman2 long time no see! Done any Tethys development as of late?

Thanks for the suggestion. Have use used django-allauth in a project? What do you see as the primary benefits?

We've been using [Python-Social-Auth](https://python-social-auth.readthedocs.io/en/latest/), which seems to address many of the same goals as django-allauth from what I can tell, but with many more supported providers (100+).

**Comment by snowman2** [comment]
Time does tend to sneak by when you don't pay attention to it. Hope all is going well on your end @swainn!

> Done any Tethys development as of late?

It has been a few years since I have used Tethys. I am glad to see it is still alive and well.

> Have use used django-allauth in a project? What do you see as the primary benefits?

I tried out both `python-social-auth` and `django-allauth` and I found `django-allauth` had the features I needed and was fairly easy to setup. Mostly personal preference.  The ~46k projects using `django-allauth` were what made me curious to try it out. That being said, `python-social-auth` also has a solid user base and is a great choice. For Tethys, the ArcMap provider in `python-social-auth` is probably good enough reason to stick with it.
